frappe/core/doctype/domain_settings/domain_settings.js

```diff
@@ -3,57 +3,11 @@
 
 frappe.ui.form.on('Domain Settings', {
 	onload: function(frm) {
-		let domains = $('<div class="domain-editor">')
-			.appendTo(frm.fields_dict.domains_html.wrapper);
-
-		if(!frm.domain_editor) {
-			frm.domain_editor = new frappe.DomainsEditor(domains, frm);
-		}
-
-		frm.domain_editor.show();
-	},
-
-	validate: function(frm) {
-		if(frm.domain_editor) {
-			frm.domain_editor.set_items_in_table();
-		}
+		frm.fields_dict.active_domains.get_data = () => {
```

Comment:
The standard pattern is to use `frm.set_df_property(fieldname, 'options', options_array)`